### ISSUE TITLE
feature(cli): `bentoml yatai update-current-context`

### ISSUE THREAD

**Issue by aarnphm** [bentoml/BentoML#3451]
### Feature request

Currently, the default context can be modified manually under `$BENTOML_HOME/.yatai.yaml`

There should be a CLI command that allow user to change this

```bash
bentoml yatai update-current-context yatai2
```

**Comment by blankRiot96** [comment]
Hey, I've got an implementation of this working, I'd like to make a PR once it's completely done
![image](https://user-images.githubusercontent.com/77634274/213719787-fcadb525-b31a-471f-8e81-296b8dae8abb.png)

**Comment by blankRiot96** [comment]
I have some issues/queries, 
- How is the data between `bento` and `bento_cli` shared, I figured to look at `bento._internals.configuration.containers`, but `_internals` tells me it isn't meant to be accessed like that from the cli?
- How does the configuration model work? Where could I store, say, a "current-context" attribute of some kind which corresponds to either ".yatai.yaml" or whatever it is the user provided

**Comment by aarnphm** [comment]
Hi there, I think that when you update current context, it should update to the default file, which is `$BENTOML_HOME/.yatai.yaml`.

> How is the data between bento and bento_cli shared?

Not sure I understand this, but they aren't relevant.

> How does the configuration model work?

Not sure I understand this also, but it doesn't have to do with yatai command.

For this feature, you need to add the `update_current_context` as a cli to https://github.com/bentoml/BentoML/blob/main/src/bentoml_cli/yatai.py

The context per yatai cluster is set in https://github.com/bentoml/BentoML/blob/main/src/bentoml/_internal/yatai_rest_api_client/config.py 

From that file, you can see we have a function call `get_config_path` that returns the path to the `.yatai.yaml`.

You probably need to implement a `set_current_context` function that reads from current .yata.yaml, update it, then dumps it back.

The CLI should be looking like:

```bash
bentoml yatai update-current-context yatai2

# Update current context to 'yatai'
```

You might also have to login with this new context as well so that `push` and `pull` will work accordingly.

**Comment by blankRiot96** [comment]
Ah, I think I'll need more understanding of this repository before I work on this
My apologies, I'll leave this for anyone else to take a stab at

**Comment by aarnphm** [comment]
No worries. Feel free to work on it as well. Sorry for not explain it clearly b4.

**Comment by OmkarMozar** [comment]
@aarnphm what is yatai2 ? is it the new context/ changed context which we are trying to update?
if yes in what form it is stored? 
**Is it the case that after login we changed yatai.yaml and now we want have a subcommand which takes this updated file and updates the context therefore login with new context as well?**

Also can assign this to me? I would like to take a shot at this one.

**Comment by aarnphm** [comment]
ah `context` is a internal feature of `bentoml yatai login` that allows us to login into multiple yatai cluster. In the yatai.yaml under `$BENTOML_HOME`, you are able to save multiple yatai login and then you can push and pull to different yatai cluster : `bentoml push bento --context c2`

`bentoml yatai update-current-context` is mainly to change the default one (_which is always set to default_) to a new context.

Tho I think this is a bit to outdated, as we might have other ideas for our cloud workflow.

**Comment by aarnphm** [comment]
Hey there, We have included this with #3991 as we migrated to the new CLI `bentoml cloud`.
